src/twist_marker.cpp

```diff
@@ -136,8 +162,16 @@ class TwistMarkerPublisher : public rclcpp::Node
     pub_->publish(marker_->getMarker());
   }
 
+  void callback_stamped(const geometry_msgs::msg::TwistStamped::ConstSharedPtr twist)
+  {
+    marker_->update(twist->twist);
```

Comment:
scratch that, you aren't even calling the update function you created above for stamped messages :D please just delete it completely